mappings/net/minecraft/world/World.mapping

```diff
@@ -11,7 +11,7 @@ CLASS none/aiw net/minecraft/world/World
 	FIELD B lootTableHandler Lnone/bbm;
 	FIELD C profiler Lnone/os;
 	FIELD D scoreboard Lnone/bdb;
-	FIELD E isRemote Z
+	FIELD E isProxy Z
```

Comment:
Probably controversial, but is there a reason why it's called isRemote aside from the fact it's the name everyone is used to?